RobloxIPFinder/Form1.cs

```diff
@@ -19,34 +21,54 @@ public Form1()
         }
 
         // Made by daymon - twitter.com/disband
+
+        // Fixed by Nick.
+        
         private void simpleButton1_Click(object sender, EventArgs e)
         {
-            int counter = 0;
-            string line;
-            string appDataFolder = Environment.GetEnvironmentVariable("LocalAppData");
-            string filePath = Path.Combine(appDataFolder, "Roblox\\logs");
-            DirectoryInfo info = new DirectoryInfo(filePath);
-            FileInfo[] files = info.GetFiles().OrderByDescending(p => p.CreationTime).ToArray();
-            foreach (FileInfo fil in files)
-            {
-                if(Convert.ToString(fil).Contains("GameStartScript"))
-                {
-                    filePath = Path.Combine(filePath, Convert.ToString(fil));
-                    break;
-                }   
-            }
-            System.IO.StreamReader file = new System.IO.StreamReader(filePath);
-            while ((line = file.ReadLine()) != null)
+
+            string LocalAppDataPath = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+
+            string FilePath = Path.Combine(LocalAppDataPath, "Roblox", "logs");
+
+            DirectoryInfo Info = new DirectoryInfo(FilePath);
+            FileInfo[] Files = Info.GetFiles().OrderByDescending(f => f.CreationTime).ToArray();

```

Comment:
Could be reduced to:
```c#
private FileInfo GetLatestLogFile()
{
    string localAppDataPath = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
    string robloxLogsPath = Path.Combine(localAppDataPath, "Roblox", "logs");

    return new DirectoryInfo(robloxLogsPath).GetFiles().OrderByDescending(file => file.LastWriteTime).First();
}
```

Reply:
Ah. Fair enough. I did all of this in my free when I did this. Ill keep this in mind tho